src/components/dls/Modal/Modal.tsx

```diff
@@ -9,8 +9,16 @@ type ModalProps = {
   triggerClassName?: string;
   open?: boolean;
   onClickOutside?: () => void;
+  shouldStopPropagation?: boolean;
 };
-const Modal = ({ children, trigger, open, onClickOutside, triggerClassName }: ModalProps) => (
+const Modal = ({
+  children,
+  trigger,
+  open,
+  onClickOutside,
+  triggerClassName,
+  shouldStopPropagation = true,
```

Comment:
since we set `shouldStopPropagation` to `true`, it will affect any component that is using `Modal`. Would that affect the functionality of the other components or all good?

Reply:
tested with other modal (advanced copy).

So the only effect is the `onClick` event will not bubble up to the parent of React component
- in the context of this PR, it will not bubble up to audioplayer. So it won't trigger audio expand
- in Advanced copy context, the onClick will not bubble up to the popover menu.

So, all good I think

Reply:
It sounds to me that shouldStopPropagation is not part of the modal scope. It should be handled in the audio player. What do you think?